std/range/package.d

```diff
@@ -7125,9 +7131,57 @@ private:
     size_t _chunkSize;
 }
 
+/// ditto
+struct Chunks(Source)
+if (isInputRange!Source && !isForwardRange!Source)
+{
+    static struct Chunk
+    {
+        private Chunks* impl;
+
+        @property bool empty() { return impl.curSizeLeft == 0 || impl.r.empty; }
```

Comment:
Doesn't dmd infer attributes on these methods, since the outer struct is a template?